srcpkgs/gnome-console/template

```diff
@@ -13,9 +13,8 @@ maintainer="oreo639 <oreo6391@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://gitlab.gnome.org/GNOME/console"
 changelog="https://gitlab.gnome.org/GNOME/console/-/raw/main/NEWS"
-#changelog="https://gitlab.gnome.org/GNOME/console/-/raw/gnome-45/NEWS"
```

Comment:
Uncomment this line and make it `https://gitlab.gnome.org/GNOME/console/-/raw/gnome-46/NEWS`, then comment the line above.